src/http/commands/help/presenter.rb

```diff
@@ -70,46 +70,66 @@ def initialize(manifest)
               self.manifest = manifest
             end
 
-            def render_html_list(data, skip_wrapper: false)
```

Comment:
we probably don't need to rename the method unless it's doing something different than before and that the new name communicates what its doing from the outside. If we rename it but keep the method maybe `#render_html` is a better name for a public method that decouples it from `_list`.